### DISCUSSION TITLE
Arcemu now supports World PvP zone activities

### DISCUSSION THREAD

**Discussion by dfighter1985** [arcemu/arcemu#457] [Announcements]
Dear All!
After about a month of work, and several commits, as of commit a3d339e  tagged as 20240208 Arcemu now supports world PvP zone activities in the following zones:

* Silithus  https://wowpedia.fandom.com/wiki/The_Silithyst_Must_Flow
* Eastern Plaguelands https://wowpedia.fandom.com/wiki/A_Game_of_Towers
* Hellfire Peninsula https://www.warcrafttavern.com/tbc/guides/hellfire-peninsula-pvp/
* Zangarmarsh https://www.warcrafttavern.com/tbc/guides/zangarmarsh-pvp/
* Terokkar Forest https://www.warcrafttavern.com/tbc/guides/terokkar-forest-pvp/
* Nagrand https://www.warcrafttavern.com/tbc/guides/nagrand-pvp/

Special thanks go to @cressidagp , @aaron02 for the help they provided.
See #347 for details of all the work done.

I'm quite sure it's not perfect, so if you find any issues feel free to report them.

Until then enjoy and have fun! :)

**Comment by cressidagp** [comment]
Awesome job @dfighter1985 !!! Its a HUGE progress.

Now go and take some months of rest & relaxation! You deserve it!

Thanks for all you're effort :)
